modules/data-entry/src/main/frontend/src/questionnaire/FormPageNavigation.jsx

```diff
@@ -76,7 +76,7 @@ function FormPageNavigation (props) {
   )
 
   // Format the title
-  let pageTitle = (page, index) => (<FormattedText variant="caption">{page?.title ? page.title : "Page " + (index + 1)}</FormattedText>)
+  let pageTitle = (page, index) => (<FormattedText variant="caption">{page?.title ?? "Page " + (index + 1)}</FormattedText>)
```

Comment:
@marta- What do we do if the page title is missing? The previous code would always replace it with "Page N", but the current code would just leave it empty, which I think is wrong.

```suggestion
  let pageTitle = (page, index) => (<FormattedText variant="caption">{page?.title || "Page " + (index + 1)}</FormattedText>)
```